.github/workflows/operator-integration-test.yml

```diff
@@ -110,3 +110,15 @@ jobs:
           cd integration-tests/manifests/cmd
           go run validate_instrumentation_vars.go default ns_instrumentation_env_variables.json
           kubectl delete instrumentation sample-instrumentation
+
+      - name: Test Annotations
+        run: |
+          kubectl apply -f integration-tests/manifests/sample-deployment.yaml
+          kubectl apply -f integration-tests/manifests/sample-daemonset.yaml
+          kubectl apply -f integration-tests/manifests/sample-statefulset.yaml
+          kubectl get pods -A
+          kubectl describe pods -n default
+          go test ./integration-tests/manifests/annotations -timeout 30m
```

Comment:
Can you add `go test -v` so it's verbose mode that way we can see each test output

Reply:
will do!